python/py-pyx/Portfile

```diff
@@ -21,20 +21,22 @@ long_description    PyX is a ${description}. It combines an abstraction of the P
 
 homepage            https://pyx-project.org/
 
-checksums           rmd160  61ec70ac7fb7b7b8a6fdac5cdb5c9328775c7018 \
-                    sha256  0fc3b00c5e7fb6f4aefbf63b95f624297dde47700a82b8b5ad6ebb346b5e4977 \
-                    size    2559840
+checksums           rmd160  f744331f506969deb31a529cdaccda8f2809d63d \
+                    sha256  4d8e3e471cd3e9a9bd13d5086cdf7c0af1b0c3f3e195e74f5f63318dc40a66d8 \
+                    size    626703
 
 # Note: py27-scapy depends on py27-pyx
-python.versions     27 37 38 39 310
+python.versions     27 37 38 39 310 311 312
 
 if {${name} ne ${subport}} {
     if {${python.version} == 27} {
         version     0.12.1
-        revision    1
+        revision    0
         checksums   rmd160  ca48a8e85c9fcd24c9050d7938ec952d410b099a \
                     sha256  e837b26a8b1c27524cf3f3dd6c0d563451249159edaa2e366d87e7143a867e8e \
                     size    561989
+
+        livecheck.type  none
```

Comment:
this should be set within the `if {${name} ne ${subport}} {` block not in the `if {${python.version} == 27} {` ; otherwhise the livecheck will still be "on" for the other subports.

Reply:
Are you sure?

See: https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/python-1.0.tcl#L575-L581

I've added a trivial patch to prove my point:
```diff
diff --git a/_resources/port1.0/group/python-1.0.tcl b/_resources/port1.0/group/python-1.0.tcl
index ca8aa79ec34..a6b06b2411e 100644
--- a/_resources/port1.0/group/python-1.0.tcl
+++ b/_resources/port1.0/group/python-1.0.tcl
@@ -577,6 +577,9 @@ pre-livecheck {
     global name subport version python._first_version
     if {[string match py-* [option name]] && ${name} ne ${subport} && ${version} eq ${python._first_version}} {
         livecheck.type  none
+        ui_info "Auto disable livecheck for ${subport}"
+    } else {
+        ui_info "Do not touch livecheck for ${subport}"
     }
 }
```
and when I had run it as `port -v livecheck 'py*-pyx'` I see the output:
```
Do not touch livecheck for py-pyx
py-pyx seems to be up to date
Do not touch livecheck for py27-pyx
Auto disable livecheck for py37-pyx
Auto disable livecheck for py38-pyx
Auto disable livecheck for py39-pyx
Auto disable livecheck for py310-pyx
Auto disable livecheck for py311-pyx
Auto disable livecheck for py312-pyx
```

Reply:
ah, true. Apparently I don't remember the code I added a while ago myself anymore ;)